test/includes/microcloud.sh

```diff
@@ -61,8 +62,9 @@ fi
 if [ -n "${SETUP_ZFS}" ]; then
   setup="${setup}
 ${SETUP_ZFS}                                            # add local disks (yes/no)
-$([ "${SETUP_ZFS}" = "yes" ] && printf "wait 300ms")    # wait for the table to populate
+$([ "${SETUP_ZFS}" = "yes" ] && printf "wait 500ms")    # wait for the table to populate
```

Comment:
I'm not so sure if this should be included here as we've seen from the preseed issue it doesn't seem to be related to the selectable table at all.

Reply:
fixed